tweets/build-2021-10-28.tweet

```diff
@@ -0,0 +1,11 @@
+Thanks to those that provide infrastructure to the Build WG:
+
+Tier 1:
+✨ @digitalocean
+✨ @Rackspace
+
+Tier 2:
+@Microsoft, @joyent, @IBM, @MacStadium, @Scaleway, @arm, @Cloudflare, @osuosl, @packethost, @intel
```

Comment:
```suggestion
@Microsoft, @joyent, @IBM, @MacStadium, @Scaleway, @arm, @Cloudflare, @osuosl, @equinixmetal, @intel
```

Both the website in the nodejs build WG repo (linked in this PR’s initial comment) and the account mentioned here point to a new brand and Twitter account, hence this suggestion.

Sorry if this is the wrong place to raise this issue - just point me to the right place if you prefer I file an issue/open a PR there!

Reply:
thanks for pointing it out :)